packages/cloudwatch/src/index.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  CloudWatchClient,
+  PutMetricDataCommand,
+} from '@aws-sdk/client-cloudwatch';
+import {
+  JobContext as PrismaJobContext,
+  JobScheduler as PrismaScheduler,
+} from '@steveojs/scheduler-prisma';
+import {
+  PendingJobs,
+  JobScheduler as SequelizeScheduler,
+} from '@steveojs/scheduler-sequelize';
+import {
+  JobAttributes as SequelizeJob,
+  JobInstance,
+} from '@steveojs/scheduler-sequelize/lib/models/job';
+
+type Job = PrismaJobContext['job'] | JobInstance;
+type JobAttributes = SequelizeJob | PrismaJobContext['job'];
+
+const client = new CloudWatchClient();
+const Namespace = 'OM-DB-Jobs';
+
+export const schedulerMetrics = (
+  scheduler: PrismaScheduler | SequelizeScheduler,
+  service: string
+) => {
+  scheduler.events.on('duration', (job: Job) => {
+    const MetricName = job!.name.toUpperCase();
```

Comment:
i kind of dislike the bang operator, i think its a code smell

why is name optional?

Reply:
I originally intended to make no changes in the schedulers themselves. 
The job type from the prisma scheduler was not exported so had to do a workaround.

I've exported it now and removed the bangs.